ci/cpu/cuspatial/build_cuspatial.sh

```diff
@@ -1,5 +1,10 @@
 set -e
 
+# If nightly build, append current YYMMDD to version
```

Comment:
This block should be in `cpu/build.sh`. I'm assuming it exists in the libcuspatial one as well, and that should be removed. We try to minimalize the `build_<package>.sh`.

Reply:
I see following error when running ci/cpu/build.sh
 ci/cpu/build.sh: line 59: ci/cpu/libcuspatial/build_libcuspatial.sh: No such file or directory

I believe this is because build_libcuspatial.sh is not executable and doesn't have #!/bin/bash denotation. build_libcuspatial.sh expects to be run as bash build_libcuspatial.sh. To me it looks like ci/cpu/build.sh is either broken or I'm missing a piece on how to run it correctly.

I would need help understanding what to do here. Cannot proceed without help.

Reply:
I think I understand. The script expect WORKSPACE variable to be set. If it's not set the script execution fails. I'll move the logic to top level script and verify by setting WORKSPACE variable

Reply:
Fixed the patch. Variable is set now in ci/cpu/build.sh. If someone calls directly build_libcuspatial.sh they would not get the desired result. Build has to be done via toplevel ci/cpu/build.sh

Reply:
Moved to ci/cpu/build.sh. Works for me locally